bin/server_stop.sh

```diff
@@ -18,15 +18,17 @@ else
 fi
 
 pidFilePath=$appdir/$PIDFILE
+PID="$(cat "$pidFilePath")"
 
 if [ ! -f "$pidFilePath" ] || ! kill -0 "$(cat "$pidFilePath")"; then
    echo 'Job server not running'
 else
   echo 'Stopping job server...'
-  PID="$(cat "$pidFilePath")"
   "$(dirname "$0")"/kill-process-tree.sh 15 $PID && rm "$pidFilePath"
   echo '...job server stopped'
 fi
 
-
-
+if [ -f "$pidFilePath" ]; then
```

Comment:
I eventually used `kill -0` again but suppressed the `stderr` otherwise printed, to keep the behaviour unchanged by my next commit